indra/tests/test_reading_files.py

```diff
@@ -0,0 +1,33 @@
+from os import path
+from indra.tools.reading.read_files import read_files, get_readers
+
+from nose.plugins.attrib import attr
+
+
+@attr('slow', 'nonpublic')
```

Comment:
We might want to add 'notravis' here because not all readers are available on Travis.

Reply:
Done.